### PR TITLE
Add support for `Stringable`

### PR DESCRIPTION
**Pull request by nlemoine**
Recent changes broke a lot of my Twig code.

Would be nice to add support for `Stringable` (mostly `Twig\Markup`).

### CODE REVIEW

File: Tests/Fixtures/html_classes.test [hunk 1]
```diff
@@ -4,9 +4,12 @@
 {{ html_classes('a', {'b': true, 'c': false}, 'd', false ? 'e', true ? 'f', '0') }}
 {% set class_a = 'a' %}
 {% set class_b = 'b' %}
-{{ html_classes(class_a, {(class_b): true})}}
```
Comment (fabpot) [inline Tests/Fixtures/html_classes.test:7]: We need to keep this test as well

**Comment by fabpot** [comment]
I've just noticed that you've submitted this PR on the wrong repo. It should be done on twigphp/twig. Can you resubmit there? Thank you.

File: Tests/Fixtures/html_classes.test (hunk 1 above)
Comment (nlemoine) [reply Tests/Fixtures/html_classes.test:7]: Indeed, will restore it on the next PR.

**Comment by nlemoine** [comment]
Sure!

**Pull request closed without merging**